website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md

```diff
@@ -0,0 +1,146 @@
+
+You can think of these Raw Vault components as LEGO bricks: they are modular and you can combine them in many different ways to build a wide variety of different, cohesive structures.
+
+Given its modular structure that requires many joins to get the specific information, Data Vault is not intended as a final, presentation layer in your data warehouse. Instead, due to the wide variety of use cases, the framework works brilliantly as the middle, integration layer of your business, serving any form of presentation layer you might have, such as wide tables, star schema, feature stores, you name it.
+
+To further accelerate the creation of these layers and prevent the repetition of the same business logic, you can make use of Business Vault as a complementary layer of your data warehouse.
+
+The Business Vault is there to fill the gaps of generic, source-data-generated Raw Vault, which often does not cover all of the business processes of your organisation. You can easily address such challenges by applying soft rules applied in this.
+
+Business Vault can also help with performance issues that can arise due to presentation layer transformations having to do lots of joins on the fly. In such scenarios, a business vault becomes a central piece for your business logic populating all of the information marts.
+
+### Should you consider Data Vault for your data warehouse?
+
+Data Vault is powerful modelling technique for middle-to-enterprise level data warehouses with the following attributes:
+
+* Integration of multiple dynamic source systems;
+* Long-term project with agile delivery requirements;
+* Auditibilty and compliance needs;
+* Preference for template based project allowing automation needs;
+* High flexibility of the data model with minimal reengineering;
+* Load performance is important, parallel loading is a must.
+
+Due to its complexity, Data Vault is not a go-to choice for:
+
+* Simple and constant systems;
+* Quick one-off solutions for experiments or short-term data warehouse projects;
+* Data warehouse layers needed for direct reporting.
+
+## dbt Cloud: the operating system for Data Vault
+
+There are many tools that can be used to implement your Data Vault project but dbt Cloud with its rich set of features provides the functionalities that make the difference by accelerating your project end to end, saving you the trouble of jumping from one tool to another.
+
+Let’s take a look at the most impactful features and explore how you can leverage them when implementing your Data Vault project.
+
+### Scalable schema
+
+Don’t Repeat Yourself (DRY) software engineering principles can help you sleep better when you are dealing with complex projects, which Data Vault most often is.
+
+dbts’ **macros **feature is a life saver in terms of templating your code. It saves you headaches due to manual errors as well as defining transformation logic in one place in case you need to change it.
```

Comment:
```suggestion
dbts’ **macros **feature is a lifesaver in terms of templating your code. It saves you headaches due to manual errors as well as defining transformation logic in one place in case you need to change it.
```